packages/terra-date-picker/src/terra-dev-site/test/date-picker/DatePickerKeyLimits.test.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
I don't think that this test page can be removed as it's not being used anymore.

Reply:
Removed https://github.com/cerner/terra-framework/pull/595/commits/edfd4d2c34f522b1a8a4742eec7e6fc6bfbd4f00